### PR TITLE
readme: add link to more examples

### PR DESCRIPTION
**Pull request by bukowa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -606,6 +606,9 @@ Examples
 A variety of example `justfile`s can be found in the
 [examples directory](https://github.com/casey/just/tree/master/examples).
 
+You can also look [here](https://github.com/search?q=path%3A%2F%28%5E%7C%5C%2F%29justfile%24%2F&type=code)
+to see how others are writing `justfile`.
+
```
Comment (casey) [inline README.md]:
```suggestion
A variety of example `justfile`s can be found in the
[examples directory](https://github.com/casey/just/tree/master/examples) and on
[GitHub](https://github.com/search?q=path%3A**%2Fjustfile&type=code).
```

**Review by casey — CHANGES_REQUESTED** [review]
Good call! See review for a suggested change.

File: README.md (hunk 1 above)
Comment (bukowa) [reply README.md]: Done!

**Review by casey — APPROVED** [review]

**Comment by casey** [comment]
This needs to be rebased on latest master.

**Comment by bukowa** [comment]
> This needs to be rebased on latest master.

Not sure if you receive notification on my pushes?
Done.

**Comment by casey** [comment]
I sometimes get notifications when someone pushes to a branch, but I don't check the PR when that happens, since a push doesn't necessarily mean the PR is ready for review.

**Comment by casey** [comment]
Looks good to me!

**Comment by bukowa** [comment]
> Looks good to me!

Damn somehow broken :D

**Comment by casey** [comment]
Whoops, that's my fault. Fixed in #2203.

**Pull request merged**